.github/workflows/setup-env/action.yml

```diff
@@ -0,0 +1,21 @@
+name: "Env setup"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v2.4.0
+      with:
+        node-version: 14
```

Comment:
Is it possible to have it working with Node 16, since it's the latest LTS?

Reply:
I don't see why not :+1: